www-servers/lighttpd/lighttpd-1.4.39.recipe

```diff
@@ -14,7 +14,7 @@ SOURCE_URI="https://download.lighttpd.net/lighttpd/releases-1.4.x/lighttpd-$port
 CHECKSUM_SHA256="bc5f910f78b8cbae3e8fe1a1d8558259f09dbe50dd3907db771404bfab27bb2e"
 PATCHES="lighttpd-$portVersion.patch"
 
-ARCHITECTURES="x86_gcc2 ?x86 ?x86_64 ?arm ?ppc"
+ARCHITECTURES="x86_gcc2 !x86 !x86_64 ?arm ?ppc"
```

Comment:
Yep, it is currently broken on x86 and x86_64. Compilation seems ok, but the link fails:
```
connections.o: In function `connection_accept':
/sources/lighttpd-1.4.39/src/connections.c:1132: undefined reference to `accept'
connections.o: In function `connection_state_machine':
/sources/lighttpd-1.4.39/src/connections.c:1585: undefined reference to `shutdown'
/sources/lighttpd-1.4.39/src/connections.c:1415: undefined reference to `shutdown'
network.o: In function `network_server_init':
/sources/lighttpd-1.4.39/src/network.c:271: undefined reference to `setsockopt'
/sources/lighttpd-1.4.39/src/network.c:258: undefined reference to `socket'
/sources/lighttpd-1.4.39/src/network.c:371: undefined reference to `connect'
/sources/lighttpd-1.4.39/src/network.c:402: undefined reference to `bind'
/sources/lighttpd-1.4.39/src/network.c:418: undefined reference to `listen'
/sources/lighttpd-1.4.39/src/network.c:249: undefined reference to `socket'
/sources/lighttpd-1.4.39/src/network.c:234: undefined reference to `socket'
/sources/lighttpd-1.4.39/src/network.c:304: undefined reference to `getaddrinfo'
/sources/lighttpd-1.4.39/src/network.c:314: undefined reference to `freeaddrinfo'
/sources/lighttpd-1.4.39/src/network.c:327: undefined reference to `gethostbyname'
/sources/lighttpd-1.4.39/src/network.c:290: undefined reference to `setsockopt'
/sources/lighttpd-1.4.39/src/network.c:282: undefined reference to `in6addr_any'
/sources/lighttpd-1.4.39/src/network.c:305: undefined reference to `gai_strerror'
/sources/lighttpd-1.4.39/src/network.c:330: undefined reference to `__h_errno'
inet_ntop_cache.o: In function `inet_ntop_cache_get_ip':
/sources/lighttpd-1.4.39/src/inet_ntop_cache.c:31: undefined reference to `inet_ntop'
configfile-glue.o: In function `config_check_cond_nocache':
/sources/lighttpd-1.4.39/src/configfile-glue.c:383: undefined reference to `inet_aton'
http-header-glue.o: In function `http_response_redirect_to_directory':
/sources/lighttpd-1.4.39/src/http-header-glue.c:141: undefined reference to `getsockname'
/sources/lighttpd-1.4.39/src/http-header-glue.c:156: undefined reference to `getnameinfo'
/sources/lighttpd-1.4.39/src/http-header-glue.c:166: undefined reference to `inet_ntop'
/sources/lighttpd-1.4.39/src/http-header-glue.c:175: undefined reference to `gethostbyaddr'
/sources/lighttpd-1.4.39/src/http-header-glue.c:178: undefined reference to `__h_errno'
/sources/lighttpd-1.4.39/src/http-header-glue.c:180: undefined reference to `inet_ntoa'
collect2: error: ld returned 1 exit status
```
I guess we just need to tweak configure.ac and wanted to dig that but haven't done it yet.
BTW, sorry for the delay, I had hit the error but forgot to keep a copy of it. And then I switched to x86_gcc2. If you have any hints I'll be happy to help "unbreak" 1.4.39 on x86_64 and x86. Hmm, if making it cross-buildable is an option, I can try it.